www/index.html

```diff
@@ -305,6 +305,20 @@ <h3>Performance settings</h3>
                                         </div>
                                     </div>
                                 </div>
+                                <br />
+                                <div class="row">
+                                    <div class="col-sm-4">
+                                        <p>Select max number of search results:</p>
+                                    </div>
+                                    <div class="col-sm-8">
+                                        <div class="range">
+                                            <label>
+                                                <input type="range" class="form-control-range" min="5" max="50" id="titleSearchRange" value="25">
+                                                <span>Value: <span id="titleSearchRangeVal"></span> (<i>increases search time</i>)</span>
```

Comment:
Maybe we should rephrase 'increases search time' to 'default is 25, higher values increase search time'?

Reply:
Yes, I'll rephrase.